<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunityFundCreatePaymentRequestDialog</class>
 <widget class="QDialog" name="CommunityFundCreatePaymentRequestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Payment Request</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutCreatePaymentRequest">
     <item>
      <widget class="QLabel" name="labelCreatePaymentRequestTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create Payment Request</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProposalHashTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Proposal:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxProposalHash"/>
     </item>
     <item>
      <widget class="QLabel" name="labelRequestedAmountTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Requested Amount (NAV):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NavcoinAmountField" name="lineEditRequestedAmount"/>
     </item>
     <item>
      <widget class="QLabel" name="labelDescriptionTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Title + URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="plainTextEditDescription"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutSubmitPaymentRequest">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonSubmitPaymentRequest">
         <property name="text">
          <string>Submit Payment Request</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacerButton">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>navcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
